Hardware/RAM/RAMInterop.cs

```diff
@@ -0,0 +1,34 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) 2011-2012 Michael Möller <mmoeller@openhardwaremonitor.org>
+// Copyright (C) 2016-2019 Sebastian Grams <https://github.com/sebastian-dev>
+// Copyright (C) 2016-2019 Aqua Computer <https://github.com/aquacomputer, info@aqua-computer.de>
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace OpenHardwareMonitor.Hardware.RAM {
+  public class Interop {
```

Comment:
I think we should follow existing structure here and put this in main RAM file under private NativeMethods class.